.github/workflows/ci.yml

```diff
@@ -42,8 +58,10 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install poetry
-      - name: Run Dagger
+      - name: aircmd core ci
         env: #TODO: Don't use GHA secrets for this
+          GITHUB_CUSTOM_TOKEN: ${{ steps.create_token.outputs.token }}
```

Comment:
Same as above, not really necessary anymore and might not even use this on other repos due to the additional setup required